.gitignore

```diff
@@ -4,3 +4,4 @@ build
 package-lock.json
 target
 Cargo.lock
+,build/
```

Comment:
Should this be .build?

Reply:
Thanks for the catch. Just updated the PR.